package/MDAnalysis/core/topologyattrs.py

```diff
@@ -2041,6 +2051,412 @@ def total_charge(group, compound='group'):
     transplants[GroupBase].append(
         ('total_charge', total_charge))
 
+    @warn_if_not_unique
+    @_pbc_to_wrap
+    @check_wrap_and_unwrap
+    def dipole_vector(group, wrap=False, unwrap=False, 
+        compound='group', center="mass"):
+        r"""Dipole vector of the group.
+
+        .. math::
+            \mathbf{\mu} = \sum_{i=1}^{N} q_{i} ( \mathbf{r_{i}} - 
+            \mathbf{r_{COM}} )
```

Comment:
Not sur if \bm works. However, only boldface the letters, not the subscripts.
```suggestion
            \bm{\mu} = \sum_{i=1}^{N} q_{i} ( \mathbf{r}_{i} - 
            \mathbf{r}_{\mathrm{COM}} )
```
(yes, I am nit-picking, but better make it pretty now when we have the chance)

Reply:
\boldsymbol might work for \mu etc. Please check the rendered docs https://mdanalysis--3842.org.readthedocs.build/en/3842/documentation_pages/core/topologyattrs.html

Reply:
yup! I think you made these comments the same time I was going through feedback.